library/src/main/java/com/github/pwittchen/reactivesensors/library/SensorNotFoundException.java

```diff
@@ -0,0 +1,12 @@
+package com.github.pwittchen.reactivesensors.library;
+
+/**
+ * Created by lucas34990 on 5/12/16.
+ */
```

Comment:
I generally, don't put "created by" comments to my projects. These information are accessible through Git.